### PR TITLE
WW-154 fixed depreciated method

### PR DESCRIPTION
**Pull request by cormorando**
https://wikia-inc.atlassian.net/browse/WW-154

### CODE REVIEW

File: extensions/3rdparty/RSS/magpierss/rss_parse.inc [hunk 1]
```diff
@@ -147,10 +147,10 @@ class MagpieRSS {
...
         $attrs = array_change_key_case($attrs, CASE_LOWER);
 
-        // check for a namespace, and split if found
+        // check for a namespace, and explode if found
         $ns = false;
         if ( strpos( $element, ':' ) ) {
-            list($ns, $el) = split( ':', $element, 2);
+            list($ns, $el) = explode( ':', $element, 2);
         }
         if ( $ns and $ns != 'rdf' ) {
```
Comment (WikiaTech) [inline extensions/3rdparty/RSS/magpierss/rss_parse.inc:155]: MAJOR Replace "and" with "&&". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2010)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 48 issues
- MAJOR 31 major
- MINOR 9 minor
- INFO 8 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. MAJOR [rss_parse.inc#L146](https://github.com/Wikia/app/blob/6982fb4443bd72c259a2ac543aa05b1b7a008bd5/extensions/3rdparty/RSS/magpierss/rss_parse.inc#L146): Remove the unused function parameter "$p". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)
2. MAJOR [rss_parse.inc#L183](https://github.com/Wikia/app/blob/6982fb4443bd72c259a2ac543aa05b1b7a008bd5/extensions/3rdparty/RSS/magpierss/rss_parse.inc#L183): Replace "or" with "||". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2010)
3. MAJOR [rss_parse.inc#L194](https://github.com/Wikia/app/blob/6982fb4443bd72c259a2ac543aa05b1b7a008bd5/extensions/3rdparty/RSS/magpierss/rss_parse.inc#L194): Replace "and" with "&&". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2010)
4. MAJOR [rss_parse.inc#L195](https://github.com/Wikia/app/blob/6982fb4443bd72c259a2ac543aa05b1b7a008bd5/extensions/3rdparty/RSS/magpierss/rss_parse.inc#L195): Replace "and" with "&&". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2010)
5. MAJOR [rss_parse.inc#L202](https://github.com/Wikia/app/blob/6982fb4443bd72c259a2ac543aa05b1b7a008bd5/extensions/3rdparty/RSS/magpierss/rss_parse.inc#L202): Replace "and" with "&&". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2010)
6. MAJOR [rss_parse.inc#L203](https://github.com/Wikia/app/blob/6982fb4443bd72c259a2ac543aa05b1b7a008bd5/extensions/3rdparty/RSS/magpierss/rss_parse.inc#L203): Replace "and" with "&&". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2010)
7. MAJOR [rss_parse.inc#L210](https://github.com/Wikia/app/blob/6982fb4443bd72c259a2ac543aa05b1b7a008bd5/extensions/3rdparty/RSS/magpierss/rss_parse.inc#L210): Replace "and" with "&&". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2010)
8. MAJOR [rss_parse.inc#L223](https://github.com/Wikia/app/blob/6982fb4443bd72c259a2ac543aa05b1b7a008bd5/extensions/3rdparty/RSS/magpierss/rss_parse.inc#L223): Replace "and" with "&&". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2010)
9. MAJOR [rss_parse.inc#L240](https://github.com/Wikia/app/blob/6982fb4443bd72c259a2ac543aa05b1b7a008bd5/extensions/3rdparty/RSS/magpierss/rss_parse.inc#L240): Replace "and" with "&&". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2010)
10. MAJOR [rss_parse.inc#L267](https://github.com/Wikia/app/blob/6982fb4443bd72c259a2ac543aa05b1b7a008bd5/extensions/3rdparty/RSS/magpierss/rss_parse.inc#L267): Remove the unused function parameter "$p". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)

**Comment by sebastianmarzjan** [comment]
LGTM

**Comment by lukaszkonieczny** [comment]
LGTM

**Pull request merged**